doc/releases/release-notes-3.2.rst

```diff
@@ -438,6 +438,9 @@ Libraries / Subsystems
 
 * SD Subsystem
 
+  * Added support for handling the :c:macro:`DISK_IOCTL_CTRL_SYNC` ioctl call.
+    this enables the filesystem api :c:func:`f_sync`.
```

Comment:
`f_sync` doesn't seem to be available (https://builds.zephyrproject.io/zephyr/pr/50447/docs/releases/release-notes-3.2.html), so better use a literal

Reply:
ah, this is a typo- meant to put `fs_sync` here